tempodb/encoding/streaming_block_test.go

```diff
@@ -173,7 +173,7 @@ func testStreamingBlockToBackendBlock(t *testing.T, cfg *BlockConfig) {
 	}
 	sort.Slice(ids, func(i int, j int) bool { return bytes.Compare(ids[i], ids[j]) == -1 })
 
-	iterator, err := backendBlock.Iterator(10 * 1024 * 1024)
+	iterator, err := backendBlock.Iterator(50 * 1024)
```

Comment:
This is a fairly large change in buffer size.  Was the previous 10MB very wrong ?

Reply:
It just force pulled every record all at once a single `Read` call. This changes it so the test actually tests the reuse of the buffer correctly.